### PR TITLE
Add radiation diffusion plotting script

### PR DESCRIPTION
**Pull request by lroberts36**
## PR Summary
Add script for making comparison plot of moment solution to analytic solution of boosted radiation diffusion problem 

## PR Checklist

- [x] Explain what you did.

### CODE REVIEW

File: scripts/python/plot_radiation_advection.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+t0 = t0p
+# Get the central position of the gaussian in the observer frame at t0 
+x0p = (0.5 - v*t0)/np.sqrt(1-v*v)
+# Lorentz factor 
+W = 1/np.sqrt(1-v*v)
+
+# Read in the files 
+base_dir = './'
+work_dir = '/'
+files = sorted(glob.glob(args.dir + 'rad_adv*.phdf'))
```
Comment (Yurlungur) [inline scripts/python/plot_radiation_advection.py:73]: You could slurp this in through argparse and let bash do the globbing if you wanted to

**Review by Yurlungur — APPROVED** [review]

File: scripts/python/plot_radiation_advection.py (hunk 1 above)
Comment (lroberts36) [reply scripts/python/plot_radiation_advection.py]: Added code to allow specifying input and output files on the command line. I am still using glob though, not sure what you had in mind for getting bash to do the globbing.

Comment (Yurlungur) [reply scripts/python/plot_radiation_advection.py]: What you've done works. I just meant you can call `plot_radiation_advection.py --files *.phdf` and they'll all be handed to `argparse`.

**Review by brryan — APPROVED** [review]
LGTM :+1:

**Comment by Yurlungur** [comment]
Sounds like you should merge when ready @lroberts36

**Pull request merged**